src/components/Dashboard/Form/Button/Button.js

```diff
@@ -2,17 +2,17 @@ import React from "react";
 import styles from "./Button.scss";
 import PropTypes from "prop-types";
 
-const Button = props => (
-  <div>
+const Button = props => {
+  return (
     <button type={props.type} className={styles.Button} onClick={props.onClick}>
-      {props.label}
+      {props.children}
     </button>
-  </div>
-);
+  );
+};
 
 Button.propTypes = {
   onClick: PropTypes.func,
-  label: PropTypes.string,
+  children: PropTypes.string,
```

Comment:
Actually, `props.children` is not string. It's native React property of props. You can read more about that here: 
https://reactjs.org/docs/composition-vs-inheritance.html :)